po/services-manager/id.po

```diff
@@ -173,8 +171,7 @@ msgstr ""
 msgid ""
 "Service %service will be %toggled and port in firewall will be %switched "
 "%link"
-msgstr ""
-"Layanan %service akan %toggled dan port pada firewall akan %diganti %tautan"
+msgstr "Layanan %service akan %toggled dan port pada firewall akan %switched %link"
```

Comment:
@lslezak This is nonsense and can never work, %s is a string but %t raises "ArgumentError: malformed format string - %t"

Reply:
It looks like the proposal code is never called anyway.